src/digital/v2_compat.rs

```diff
@@ -140,10 +144,12 @@ mod tests {
         }
     }
 
+    #[cfg(feature = "unproven")]
     struct NewToggleablePinConsumer<T: v2::ToggleableOutputPin> {
         _pin: T,
     }
 
+    #[cfg(feature = "unproven")]
```

Comment:
... and this

Reply:
`v2::ToggleableOutputPin` is `unproven`, and `NewToggleablePinConsumer` depends on it, so this test wouldn't build without the flag anyway.